components/link/metadata/link.yml

```diff
@@ -1,34 +1,50 @@
+id: link-primary
 name: Link
 SpectrumSiteSlug: https://spectrum.adobe.com/page/link/
+sections:
+  - name: Migration Guide
+    description: |
+      ### Subtle variant
+      Subtle variant was removed. Please use Secondary.
```

Comment:
I believe `quiet` is the substitution for `subtle`, but I could be wrong.

Reply:
Fixed.